packages/desktop-client/src/components/reports/reports/CustomReportListCards.tsx

```diff
@@ -47,9 +47,9 @@ export function CustomReportListCards({
   const payees = usePayees();
   const accounts = useAccounts();
   const categories = useCategories();
-  const { isNarrowWidth } = useResponsive();
   const [_firstDayOfWeekIdx] = useLocalPref('firstDayOfWeekIdx');
   const firstDayOfWeekIdx = _firstDayOfWeekIdx || '0';
+  const { isNarrowWidth } = useResponsive();
```

Comment:
nit - does this need to be moved?

Reply:
Sorry, I don't quite follow. Do you mean moved to a different spot in the file, or moved to another file? It passes lint so I don't think it's the former, and if it's the latter I'm using it here to calculate element flex-basis in desktop vs mobile screen sizes

Reply:
@carkom Let me know if there's anything I need to do to get this merged